### ISSUE TITLE
bcnc fails to run

### ISSUE THREAD

**Issue by paper42** [void-linux/void-packages#46723]
### Is this a new report?

Yes

### System Info

Void 6.1.55_2 x86_64-musl GenuineIntel uptodate rFFFFFFFFFFFFFFFFF

### Package(s) Affected

bCNC-0.9.14.307_4

### Expected behaviour

runs

### Actual behaviour

```
Traceback (most recent call last):
  File "/usr/bin/bcnc", line 33, in <module>
    sys.exit(load_entry_point('bCNC==0.9.14.307', 'console_scripts', 'bCNC')())
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/bin/bcnc", line 25, in importlib_load_entry_point
    return next(matches).load()
           ^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/importlib/metadata/__init__.py", line 205, in load
    module = import_module(match.group('module'))
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/lib/python3.12/importlib/__init__.py", line 90, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1381, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1354, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1325, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 929, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 994, in exec_module
  File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
  File "/usr/lib/python3.12/site-packages/bCNC/__main__.py", line 50, in <module>
    import Utils
  File "/usr/lib/python3.12/site-packages/bCNC/Utils.py", line 41, in <module>
    if getattr( sys, 'frozen', False ):
                ^^^
NameError: name 'sys' is not defined. Did you forget to import 'sys'?
```

### Steps to reproduce

1. `xbps-install bcnc`
2. `bcnc`

**Comment by Luciogi** [comment]
it is fixed in newer version 0.9.15
but it need dependency `svgelements` which is not available in void repos

**Comment by paper42** [comment]
https://github.com/void-linux/void-packages/pull/46817
